src/lwc/gsChecklist/gsChecklist.js

```diff
@@ -0,0 +1,64 @@
+import { LightningElement, api, track } from 'lwc';
+import gsChecklistArrowRightAlterText from '@salesforce/label/c.gsChecklistArrowRightAlterText';
+import gsChecklistArrowDownAlterText from '@salesforce/label/c.gsChecklistArrowDownAlterText';
+
+export default class gsChecklist extends LightningElement {
```

Comment:
we need a class jsDoc here

Reply:
Thanks for checking for function comments @dcaballeroSf